docs/doxylink_setup.md

```diff
@@ -0,0 +1,69 @@
+<!--
+=====================================================================================
+ C O P Y R I G H T
+-------------------------------------------------------------------------------------
+ Copyright (c) 2022 by Robert Bosch GmbH. All rights reserved.
```

Comment:
Happy new year :) We are in 2023. We could start updating the new copyright headers accordingly :)